### PR TITLE
Feature: Back + Save & Exit for app form

### PR DESCRIPTION
**Pull request by dimak1**

### CODE REVIEW

**Comment by hamzajaved** [comment]
You will need to rebase on develop. I've made a few changes to the form so you might run into conflicts

**Comment by dimak1** [comment]
This is how it looks
form-buttons

File: app/containers/Forms/Form.tsx [hunk 1]
```diff
@@ -19,6 +20,7 @@ import ProductionFields from './ProductionFields';
 interface Props {
   query: Form_query;
   onComplete?: (e: IChangeEvent) => void;
+  onBack?: () => void;
```
Comment (matthieu-foucault) [inline app/containers/Forms/Form.tsx]: This prop (and `onComplete` should probably be required)

Comment (dimak1) [reply app/containers/Forms/Form.tsx]: Updated to required

**Comment by dimak1** [comment]
Added tests

File: app/tests/containers/Forms/Form.test.tsx [hunk 2]
```diff
@@ -100,13 +106,19 @@ describe('Form', () => {
...
+              }
             }
           }
         }}
       />
     );
     expect(r).toMatchSnapshot();
+    expect(r.find('.form-submit a').length).toBe(1);
+    expect(r.find('Button').length).toBe(2);
+    expect(r.find('Button.btn-secondary').text()).toBe('Back');
```
Comment (matthieu-foucault) [inline app/tests/containers/Forms/Form.test.tsx]: Looks good. I'll merge this now, but I'll rewrite it in a separate unit test (using shallow render) in #205 , as I changed this file to an integration test

**Pull request merged**